libraries/db/include/graphene/db/index.hpp

```diff
@@ -402,14 +402,17 @@ namespace graphene { namespace db {
             DerivedIndex::remove(obj);
          }
 
-         virtual const object& insert( object&& obj )override
-         {
-             const auto& res = DerivedIndex::insert(std::move(obj));
-             for( const auto& item : _sindex )
-                item->object_inserted( res );
-             on_add(res);
-             return res;
-         }
+         // Note: Implementing insert function here will break the
```

Comment:
Please remove commented code. If we need to know how this was done we can refer to specific commit, pull request, etc. No need to keep it.